app/upgrades/v8/legacy_subspaces.go

```diff
@@ -42,18 +49,22 @@ func getLegacySubspaces(paramsKeeper *paramskeeper.Keeper) paramstypes.Subspace
 			keyTable = govv1.ParamKeyTable() //nolint:staticcheck
 		case crisistypes.ModuleName:
 			keyTable = crisistypes.ParamKeyTable() //nolint:staticcheck
-		// ibc types
-		case ibctransfertypes.ModuleName:
-			keyTable = ibctransfertypes.ParamKeyTable()
-		case icahosttypes.SubModuleName:
-			keyTable = icahosttypes.ParamKeyTable()
-		case icacontrollertypes.SubModuleName:
-			keyTable = icacontrollertypes.ParamKeyTable()
 		// wasm
 		case wasmtypes.ModuleName:
 			keyTable = wasmtypes.ParamKeyTable() //nolint:staticcheck
-		default:
+
+		// these modules have not migrated away from x/params
+		case halving.DefaultParamspace,
+			ibcexported.ModuleName,
+			ibctransfertypes.ModuleName,
+			icacontrollertypes.SubModuleName,
+			icahosttypes.SubModuleName,
+			lscosmostypes.ModuleName,
+			oracletypes.ModuleName,
+			packetforwardtypes.ModuleName:
```

Comment:
I think we miss `builder.ModuleName` for pob - should have another case for modules that need to `continue` because already on v47 SDK type of params.

Reply:
param subspace was not being used for POB, so removed it from there.